src/status_im/contexts/profile/profiles/view.cljs

```diff
@@ -173,68 +174,81 @@
-        :type                :grey
-        :background          :blur
-        :icon-only?          true
-        :on-press            set-show-profiles
-        :disabled?           processing
-        :accessibility-label :show-profiles}
-       :i/multi-profile]]
-     [rn/scroll-view
-      {:keyboard-should-persist-taps :always
-       :style                        {:flex 1}}
-      [quo/profile-card
-       {:name                name
-        :customization-color (or customization-color :primary)
-        :profile-picture     profile-picture
-        :card-style          style/login-profile-card}]
-      [standard-authentication/password-input
-       {:shell?              true
-        :blur?               true
-        :on-press-biometrics (when (= auth-method constants/auth-method-biometric)
-                               (fn []
-                                 (rf/dispatch [:biometric/authenticate
-                                               {:on-success #(rf/dispatch
-                                                              [:profile.login/biometric-success])
-                                                :on-fail    #(rf/dispatch
-                                                              [:profile.login/biometric-auth-fail
-                                                               %])}])))
-        :default-password    password}]]
-     [quo/button
-      {:size                40
-       :type                :primary
-       :customization-color (or customization-color :primary)
-       :accessibility-label :login-button
-       :icon-left           :i/unlocked
-       :disabled?           (or (not sign-in-enabled?) processing)
-       :on-press            login-multiaccount
-       :container-style     {:margin-bottom (+ (safe-area/get-bottom) 12)}}
-      (i18n/label :t/log-in)]]))
+  []
+  (let [password-value (reagent/atom nil)]
+    (fn [{:keys [set-show-profiles]}]
+      (let [{:keys [processing password]}              (rf/sub [:profile/login])
+            {:keys [key-uid name customization-color]} (rf/sub [:profile/login-profile])
+            sign-in-enabled?                           (rf/sub [:sign-in-enabled?])
+            profile-picture                            (rf/sub [:profile/login-profiles-picture key-uid])
+            auth-method                                (rf/sub [:auth-method])
+            login-multiaccount                         #(rf/dispatch [:profile.login/login])
+            on-change-password                         (fn [value]
+                                                         (reset! password-value value)
+                                                         (when (not= value
+                                                                     (security/safe-unmask-data
+                                                                      password))
+                                                           (rf/dispatch [:set-in
```

Comment:
These type of events `:set-in`, `:set-once`, etc are deprecated. Although not easy to realize that 🤷🏼 

https://github.com/status-im/status-mobile/blob/bfed36d19f89ad196f4089526ee33f5e822c6f9c/src/legacy/status_im/events.cljs#L220-L222

Reply:
They are still being used here @mmilad75. Maybe you forgot to push?